.github/workflows/pr_unit_test.yaml

```diff
@@ -0,0 +1,27 @@
+name: pr_unit_test
+on:
+    pull_request:
+        branches:
+            - main
+        types: [opened, reopened, synchronize]
+
+jobs:
+    ktlint:
```

Comment:
change job name to something else because you can't have two jobs with the same name .

Reply:
check conflict here : https://github.com/scribe-org/Scribe-Android/blob/a5ce35e85e8c82a6c4442d7e31a1958733a3029c/.github/workflows/pr_ktlint_detekt.yaml#L9